openid-4-verifiable-presentations-1_0.md

```diff
@@ -572,11 +572,11 @@ When a VP Token is returned, the respective response MUST include the following
 : REQUIRED. JSON String or JSON object that MUST contain a single Verifiable Presentation or an array of JSON Strings and JSON objects each of them containing a Verifiable Presentations. Each Verifiable Presentation MUST be represented as a JSON string (that is a Base64url encoded value) or a JSON object depending on a format as defined in Appendix A of [@!OpenID.VCI].  When a single Verifiable Presentation is returned, the array syntax MUST NOT be used.  If Appendix A of [@!OpenID.VCI] defines a rule for encoding the respective Credential format in the Credential Response, this rules MUST also be followed when encoding Credentials of this format in the `vp_token` response parameter. Otherwise, this specification does not require any additional encoding when a Credential format is already represented as a JSON object or a JSON string.
```

Comment:
Here, vp_token is still required, above it is optional.

Reply:
umm.. in the new query language, vp_token would still be present, just it's structure would change, no? PR #266 seems to be silent on the response?

Reply:
And this definition MUST be relaxed as well when we want to allow a different structure with a new query language.